apollo-gradle-plugin-incubating/src/main/kotlin/com/apollographql/apollo/gradle/api/ApolloExtension.kt

```diff
@@ -8,6 +8,7 @@ import org.gradle.api.provider.Property
 interface ApolloExtension: CompilerParams {
 
   val compilationUnits: DomainObjectCollection<CompilationUnit>
```

Comment:
Maybe remove this from the public API now that we have `onCompilationUnits` ? That'll make it easier to change implementation later if needed

Reply:
Good idea 🙌 I was going to ask that actually. Android plugin and Gradle provides both time to time. Especially Gradle is full of overloaded functions doing the same thing 🙃 That's why I was not sure. Will make it private

Reply:
I think it would make sense to have it public if the collection were writable by users but it's not, we don't want anyone to write something in there.